File: Practice/09/Python/9.py
h1, m1 = map(int, input().split(':'))
h2, m2 = map(int, input().split(':'))
0 <= h1, h2 <= 23
0 <= m1, m2 <= 59
time1 = h1 + (m1/60)
time2 = h2 + (m2/60)
if (0 <= (time2 - time1) <= 0.25):
    print("Встреча состоялась.")
else:
    print("Встреча не состоялась.")

File: Practice/09/C++/9/9/9.cpp
#include <iostream>
using namespace std;

int main() {
    setlocale(LC_ALL, "Rus");
   
    double h1, h2, m1, m2, time1, time2;
    cin >> h1 >> ":" >> m1;
    cin >> h2 >> ":" >> m2;
    time1 = h1 + (m1 / 60);
    time2 = h2 + (m2 / 60);
    if ((0 <= time2 - time1) && (time2 - time1 <= 0.25)){
        cout << "Встреча состоялась";
    }
    else {
        cout << "Встреча не состоялась";
    }
}